packages/nextjs/pages/api/og.tsx

```diff
@@ -0,0 +1,205 @@
+import { NextRequest } from "next/server";
+import { ImageResponse } from "@vercel/og";
+import { Address, createPublicClient, http } from "viem";
+import { mainnet } from "viem/chains";
+
+export const publicClient = createPublicClient({
+  chain: mainnet,
+  transport: http(),
+});
```

Comment:
I think this should be dynamic right base on chain selected ? 

We could use same logic from https://github.com/BuidlGuidl/abi.ninja/pull/81/files#r1578770370 to get full viem chain object from search params. 

Also instead of using just `http()` we could use `http(alchemyRPCURL)` logic in above code

Reply:
I did it a bit different than you wanted: [https://github.com/BuidlGuidl/abi.ninja/pull/90/commits/8769b9d7ceed4e4c1266aa400491be1c55193260](https://github.com/BuidlGuidl/abi.ninja/pull/90/commits/8769b9d7ceed4e4c1266aa400491be1c55193260)

Let me know if you would like me to change it to the way you have suggested!

Reply:
Ohh looks good!! 

Just pushed https://github.com/BuidlGuidl/abi.ninja/pull/90/commits/b557842fe1bf5e902dad8281dd74a305aba0593f to use alchemy rpc. (which is a bit faster) instead of public rpc